src/bgw/scheduler.c

```diff
@@ -486,12 +491,31 @@ ts_populate_scheduled_job_tuple(ScheduledBgwJob *sjob, Datum *values)
 }
 #endif
 
+static int
+cmp_next_start(const void *left, const void *right)
+{
+	const ListCell *left_cell = *((ListCell **) left);
+	const ListCell *right_cell = *((ListCell **) right);
+	ScheduledBgwJob *left_sjob = lfirst(left_cell);
+	ScheduledBgwJob *right_sjob = lfirst(right_cell);
```

Comment:
```suggestion
	ScheduledBgwJob *left_sjob = lfirst_node(ScheduledBgwJob, left_cell);
	ScheduledBgwJob *right_sjob = lfirst_node(ScheduledBgwJob, right_cell);
```

Reply:
this doesn't work as ScheduledBgwJob is not a Postgres node type (There is no T_ScheduledBgwJob).

Reply:
Ah. OK.